$MirOS: ports/x11/gnome/vfs2/patches/patch-configure_in,v 1.1 2005/10/15 17:23:49 bsiegert Exp $
--- configure.in.orig	Tue Apr 10 14:03:46 2007
+++ configure.in	Wed Jun 20 00:20:05 2007
@@ -164,7 +164,10 @@ AC_STRUCT_ST_BLOCKS
 
 dnl Volume monitor stuff			     
 AC_CHECK_FUNCS(setmntent endmntent hasmntopt getmntinfo)
-AC_CHECK_HEADERS(mntent.h sys/mnttab.h sys/vfstab.h sys/cdio.h sys/mount.h sys/mntctl.h sys/vfs.h sys/vmount.h sys/sysctl.h fstab.h fnmatch.h util.h sys/sysmacros.h)
++AC_CHECK_HEADERS(sys/param.h)
+AC_CHECK_HEADERS(mntent.h sys/mnttab.h sys/vfstab.h sys/cdio.h sys/mount.h sys/mntctl.h sys/vfs.h sys/vmount.h sys/sysctl.h fstab.h fnmatch.h util.h sys/sysmacros.h, [], [], [#ifdef HAVE_SYS_PARAM_H
+#include <sys/param.h>
+#endif])
 
 
 AC_PATH_PROG(GCONFTOOL, gconftool-2, no)
